### ISSUE TITLE
{Mechanic} Boss Chests

### ISSUE THREAD

**Issue by Vafnar** [AngelArenaAllstars/suggestions#261]
TL;DR: A system where loot is distributed based on how much damage you did to a chest that spawns after the boss died.

For some bosses a teamfight may follow, or the loot could be destroyed. This is suppose to be a way for bosses to drop and distribute loot. instead of the loot dropping on the ground leaving it for free rain amongst those that are there, why not have a chest drop that you have 10 seconds to kill, depending on the damage dealt different items will be sent into your stash.
For example if you deal 10K damage on a chest that has 10M health, you get the items that are commonly dropped, but if you deal 3M damage on the same chest you get the Rarely dropped items.

**Comment by Haganeko** [comment]
Things don't drop, we have ngp for that. What does this accomplish tha ngp doesn't?
